cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -386,6 +386,38 @@ function DownloadAndInstall-KubernetesBinaries {
   Remove-Item -Force -Recurse $tmp_dir
 }
 
+# Downloads the csi-proxy binaries from kube-env's CSI_PROXY_TAR_URL and
+# puts them in a subdirectory of $env:K8S_DIR.
+#
+# Required ${kube_env} keys:
+#   CSI_PROXY_TAR_URL
+function DownloadAndInstall-CSIProxyBinaries {
+
+  $tmp_dir = 'C:\k8s_tmp'
+  New-Item -Force -ItemType 'directory' $tmp_dir | Out-Null
+
+
+  if (Test-IsTestCluster $kube_env) {
+
+    if (ShouldWrite-File ${env:NODE_DIR}\csi-proxy.exe) {
+      # Here we use $env to set up url first, later will change to use kube_env.
+      $urls = ${env.CSI_PROXY_TAR_URL}
+      $filename = Split-Path -leaf $urls[0]
+      if ($kube_env.ContainsKey('CSI_PROXY_TAR_HASH')) {
+        $hash = ${kube_env}['CSI_PROXY_TAR_HASH']
+      }
+      MustDownload -File -Hash $hash -OutFile $tmp_dir\$filename -URLs $urls
```

Comment:
`MustDownload -File` -> `MustDownload-File`

Also, does this line work is `$hash` is not defined (the block at line 407 was not entered)?